src/zamp_clear_zamp_store.prog.abap

```diff
@@ -0,0 +1,8 @@
+*&---------------------------------------------------------------------*
+*& Report zamp_clear_zamp_store
+*&---------------------------------------------------------------------*
+*&
+*&---------------------------------------------------------------------*
+REPORT zamp_clear_zamp_store.
```

Comment:
Maybe we can extend that in the future to a admin report e.g. delete for specific scenarios.
We should also think about what can be the dangers to run such a program in production :)

Reply:
We may also remove it later when the zamp_store is in a solid state. 
I created an issue for that topic (https://github.com/Goala/abap-metrics-provider/issues/51)